docs/2.develop/contracts/anatomy.md

```diff
@@ -154,7 +134,13 @@ The `default` method defines the default parameters to initialize the contract.
           url="https://github.com/near-examples/donation-js/blob/master/contract/src/contract.ts"
           start="6" end="8" />
 
-  In JavaScript, the default value is defined by the class parameters definition.
+  🌐 In JavaScript, the default state is defined by the initialization parameters in the class definition.
+
+
+:::caution
+In Javascript you **must always** give a value to **all parameters** of the class.
```

Comment:
It would be nice for this to explain shortly why this is necessary, and link to storage/state section.
```suggestion
In Javascript you **must always** give a value to **all parameters** of the class. This ensures that values, when deserialized are attributed to their intended types ([read more on state and deserialization](./serialization.md)).
```